Dockerfile

```diff
@@ -3,22 +3,21 @@ FROM ubuntu:16.04
 ENV DEBIAN_FRONTEND noninteractive
 
 # File Author / Maintainer
-MAINTAINER Yun-Fang Juan yunfang@chanzuckerberg.com
+MAINTAINER Charles De Bourcy cdebourcy@chanzuckerberg.com
```

Comment:
you could also use idseq-tech@chanzuckerberg.com

Reply:
Thanks, will update that.